opnode/rollup/derive/payload_attributes.go

```diff
@@ -271,7 +271,7 @@ func SortedAndPreparedBatches(batches []*BatchData, epoch, blockTime, minL2Time,
 	return out
 }
 
-func L1InfoDepositBytes(l1Info L1Info) (l2.Data, error) {
+func L1InfoDepositBytes(l1Info L1Info) (hexutil.Bytes, error) {
```

Comment:
Changed to avoid import cycle (`l2` depends on `derive` to implement `BlockReference` API, but `derive` depended on `l2` for types definition)

Reply:
That's fine, wasn't supposed to be `l2.Data` in the first place. `l2.Data` is only there to describe the `DATA` type in JSON RPC, used in the transactions list for each element.